src/algebra/algebra/spectrum.lean

```diff
@@ -212,6 +213,25 @@ theorem preimage_units_mul_eq_swap_mul {a b : A} :
   (coe : Rˣ → R) ⁻¹' σ (a * b) = coe ⁻¹'  σ (b * a) :=
 by { ext, exact unit_mem_mul_iff_mem_swap_mul, }
 
+section star
+
+variables [star_add_monoid R] [star_ring A] [star_module R A]
+
+lemma star_mem_resolvent_set_star {r : R} {a : A} (h : r ∈ resolvent_set R a) :
+  star r ∈ resolvent_set R (star a) :=
```

Comment:
I wonder if this might be more useful:
```suggestion
lemma star_mem_resolvent_set_iff {r : R} {a : A} :
  star r ∈ resolvent_set R a ↔ r ∈ resolvent_set R (star a) :=
```